jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/processinstance/JPAProcessInstanceManager.java

```diff
@@ -69,6 +71,55 @@ public void setKnowledgeRuntime(InternalKnowledgeRuntime kruntime) {
         this.kruntime = kruntime;
     }
 
+    @Override
+    public Collection<ProcessInstance> loadKnowledgeRuntimeProcessInstances() {
+        InternalRuntimeManager manager = (InternalRuntimeManager) kruntime.getEnvironment().get(EnvironmentName.RUNTIME_MANAGER);
+        List<Long> ids = Collections.emptyList();
```

Comment:
ids will take a value if the "if or else" clause, so this assignment is useless.

Reply:
done